R/transformCounts.R

```diff
@@ -349,6 +357,15 @@ setMethod("relAbundanceCounts",signature = c(x = "SummarizedExperiment"),
     return(mat)
 }
 
+.calc_rank <- function(mat, pseudocount, ...){
+    mat <- .apply_pseudocount(mat, pseudocount)
+    # For every sample, finds ranks of taxa.
+    # Column-wise, NAs are kept as NAs, and ties get the minimum rank value
+    # as.matrix, because otherwise error occurs if mat is, e.g., DelayedArray
+    mat <- apply(as.matrix(mat), MARGIN = 2, FUN = rank, na.last = "keep", ties.method = "min")
```

Comment:
`colRanks` from `DelayedMatrixStats` should be much faster.

Reply:
Changed